src/stores/models/Option.test.js

```diff
@@ -0,0 +1,303 @@
+import { observable } from 'mobx';
+import { Token, Phases, OracleStatus } from 'constants';
+import Option from './Option';
+const localStorageMock = {
+  getItem: jest.fn(),
+  setItem: jest.fn(),
+  clear: jest.fn(),
+};
+const navigatorMock = {};
+global.navigator = navigatorMock;
+global.localStorage = localStorageMock;
+
+const { QTUM, BOT } = Token;
+const { BETTING, VOTING, FINALIZING } = Phases;
+
+describe('stores/models/Option', () => {
+  const addr = {
+    address: 'qSu4uU8MGp2Ya6j9kQZAtizUfC82aCvGT1',
+    qtum: 2000,
+    bot: 100,
+  };
+  const app = observable({
+    sortBy: 'ASC',
+    wallet: { addresses: [addr] },
+    global: { syncBlockNum: 0 },
+    refreshing: false,
+    ui: { location: 0 },
+    eventPage: { selectedOptionIdx: 0 },
+  }); // mock the appstore
+  let store;
+
+  beforeEach(() => store = null);
+
+  it('Constructor 1: centralized oracle unconfirmed', () => {
+    const name = 'cons test 1';
+    const i = 0;
+    const oracle = observable({
```

Comment:
since you are converting it to an `observable`, does the Option model need to be converted to Mobx (tag vars as @observable)?

Reply:
also, seems like there are reusable parts of the `oracle` object. in the `describe('constructor(), () => {}` you can create this oracle and just change certain values for certain tests.

```
describe('constructor()', () => {
  let oracle;
  beforeEach(() => {
    oracle = observable({
      txid: 0,
      amounts: [99, 100],
      consensusThreshold: 100,
      address: '02e91962156da21fae38e65038279c020347e4ff',
      topicAddress: '4044f951857f2885d66d29a475235dacdaddea84',
      resultSetEndTime: 10,
      endTime: 20,
      options: [name, 'cons test 2'],
      token: QTUM,
      phase: BETTING,
      optionIdxs: [0, 1],
      status: OracleStatus.VOTING,
    });
  });

  it('Constructor 2: centralized oracle confirmed', () => {
    oracle.unconfirmed = false;
  });
};
```